test/models/user_test.rb

```diff
@@ -1,7 +1,12 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  context 'user basic info' do
+    setup do
```

Comment:
`setup` is for sharing code between tests and in general shouldn't contain tests. I would create explicit tests for the email/password stuff